linera-execution/src/runtime.rs

```diff
@@ -60,6 +60,8 @@ pub(crate) struct ExecutionRuntime<'a, C, const WRITABLE: bool> {
     bytes_read: AtomicU64,
     /// The total size being written
     bytes_written: AtomicU64,
+    /// The total size being written
+    change_stored_size: AtomicI32,
```

Comment:
(I find it odd that each field in `ExecutionRuntime` is locked separately. I don't think we should ever have multiple threads accessing different fields simultaneously?)

Reply:
Well, I think that the point of atomics is that they are not lock-based but use Compare and Swap operations in the CPU. Maybe we could unify that stuff, but I see that as part of other works.

Reply:
@afck Yes totally. We want to change this with @jvff